integr8ly/integreatly-operator
# Update Apicurito to version 7.6.0

# Description
Update Apicurito to version 7.6.0 from 7.5.0
https://issues.redhat.com/browse/INTLY-5732

# How to test
- Install RHMI
- Open Apicurito via route
- Verify you can create a new API
- Verify you can generate and download a Camel Project (zip file) from the newly created API. Link is in the top right of web app
